adapters/appnexus/appnexus.go

```diff
@@ -119,26 +119,22 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.E
 	// For this all impressions in  request should belong to the same pod
 	// If impressions number per pod is more than maxImpsPerReq - divide those imps to several requests but keep pod id the same
 	// If  adpodId feature disabled and impressions number per pod is more than maxImpsPerReq  - divide those imps to several requests but do not include ad pod id
-	if isVIDEO == 1 && *adPodId {
+	if isVIDEO == 1 && *shouldGenerateAdPodId {
```

Comment:
Can we move this code to a different function?

Reply:
https://github.com/prebid/prebid-server/pull/2632/commits/c1ec4ac2d47ee9f35d2e88ef17d6a0e7389cf396 extracts logic into separate function